crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -268,14 +269,23 @@ def run_installer
 
       def restore_database
         Rails.logger.debug "Restoring Crowbar database"
-        migrate_database(:before)
+        if ActiveRecord::Migrator.get_all_versions.include? @migration_level
+          migrate_database(:before, @migration_level)
+        else
+          Rails.logger.error "Cannot migrate to #{@migration_level}. Migration unknown"
+          set_failed
+          @status[:restore_database] = {
+            status: :not_acceptable,
+            msg: I18n.t("backups.index.restore_database_failed")
+          }
```

Comment:
heh, I think this was forgotten when we introduced the second migrate_database call... thanks again